Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevels.kt

```diff
@@ -1,7 +1,11 @@
 package de.rki.coronawarnapp.risk
 
 import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import com.google.android.gms.nearby.exposurenotification.ExposureWindow
+import de.rki.coronawarnapp.risk.result.AggregatedRiskResult
+import de.rki.coronawarnapp.risk.result.RiskResult
 import de.rki.coronawarnapp.server.protocols.internal.AttenuationDurationOuterClass
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
 
 interface RiskLevels {
```

Comment:
I'm sure the name of this interface was set long before you took over but is there a chance to adjust it? In the end, that's the actual Risk Calculation, isn't it? I don't know what RiskLevels is until I look into the code. Judging just from the name, I would have expect this to be an enum or a collection of constants.

Reply:
I was thinking about refactoring all the parts of the new risk level calculation into their own classes but we decided to postpone this untill a later date when more of the code was completed and we had a better image of if it would be necessary. Maybe @d4rken can shed some more light on the naming choices here.

Reply:
AFAIK, @chris-cwa  picked the name (ping). I don't see why we couldn't rename this if desired.

Reply:
I think a simple name-change-refactoring shouldn't be too much trouble 😉

Reply:
sure, well discuss some more and change it to something more fitting in a later pr 👍